app/controllers/application_controller.rb

```diff
@@ -8,6 +8,14 @@ class ApplicationController < ActionController::Base
   protected
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.for(:sign_up) << :username
+    devise_parameter_sanitizer.for(:sign_up) {
+      |u| u.permit(:username, :email, :password, :password_confirmation, :remember_me)
+                                              }
```

Comment:
`end` at 13, 46 is not aligned with `devise_parameter_sanitizer.for(:sign_up) {` at 11, 4.